workflow-core/src/commonMain/kotlin/com/squareup/workflow1/RuntimeConfig.kt

```diff
@@ -161,6 +169,8 @@ public enum class RuntimeConfigOptions {
       //     DRAIN_EXCLUSIVE_ACTIONS,
       //   )
       // ),
+
+      ALL(RuntimeConfigOptions.entries.toSet())
```

Comment:
It seems to me like it would be a good idea to have a config that turns on all the flags (for testing, but also for green-field uses of the runtime), but if you'd rather I add explicit configs for this new flag I can do that too.

Reply:
Added a comment indicating the difference in intention between this and other enum entries that might happen to also contain all entries when they're introduced.